static/app/views/monitors/components/overviewTimeline/monitorEnvironmentLabel.tsx

```diff
@@ -14,13 +17,39 @@ interface Props {
   monitorEnv: MonitorEnvironment;
 }
 
+const userNotifiedDisplay = {
+  label: t(
+    'This environment is likely broken due to being in an error state for multiple days.'
+  ),
+  icon: <IconFix color="subText" />,
+  color: 'subText' as ColorOrAlias,
+};
+
+const envMutedDisplay = {
+  label: t(
+    'This environment is likely broken due to being in an error state for multiple days. It has been automatically muted.'
+  ),
+  icon: <IconMute color="subText" />,
+  color: 'subText' as ColorOrAlias,
+};
```

Comment:
Let's add a quick type for these

Reply:
```tsx
interface StatusNotice {
  label: React.ReactNode,
  icon: React.ReactNod,
  color: ColorOrAlias,
}
```

Reply:
Yeah I think that should use the `StatusNotice` (maybe there's a better name) type too